client/src/App.js

```diff
@@ -14,6 +14,9 @@ function App() {
       <GlobalStyle />
       <Router>
         <Switch>
+          <Route path="/tours/:id/edit">
+            <AddTour concurrentTour />
```

Comment:
```suggestion
            <AddTour />
```
Is there still need for this?

Reply:
Just forgot to remove this one. 😄